simulatingrisk/hawkdovemulti/simrisk_batch.slurm

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+#SBATCH --job-name=simrisk       # job short name 
+#SBATCH --nodes=1                # node count
+#SBATCH --ntasks=1               # total number of tasks across all nodes
+#SBATCH --cpus-per-task=20       # cpu-cores per task 
+#SBATCH --mem-per-cpu=600M       # memory per cpu-core
+#SBATCH --time=02:00:00          # total run time limit (HH:MM:SS)
+#SBATCH --mail-type=begin        # send email when job begins
+#SBATCH --mail-type=end          # send email when job ends
+#SBATCH --mail-type=fail         # send email if job fails
+#SBATCH --mail-user=EMAIL
+
+module purge
+module load anaconda3/2023.9
+conda activate simrisk
+
+# change working directory for data output
+cd /scratch/network/<USER>/simrisk
+
+# test run: one iteration, max of 125 steps, no progress bar
+# (typically completes in less than 15 minutes)
+simrisk-hawkdovemulti-batchrun --iterations 1 --max-step 125 --no-progress
+
+# longer run: one iteration, max of 125 steps, no progress bar
+# (typically completes in less than 15 minutes)
```

Comment:
Either remove this line or provide an estimate of how long 10 iterations typically takes
```suggestion
```

Reply:
I haven't figure out that estimate yet for the current config (I tweaked the parameter set), but I probably need to for my own batch running anyway - do you think it's worth including some time estimates here?

Reply:
Not really, unless the extrapolation of 150 minutes (10 * 15) is a really off